packages/now-node/src/index.ts

```diff
@@ -470,31 +471,47 @@ export async function startDevServer(
 async function doTypeCheck({
   entrypoint,
   workPath,
+  meta = {},
 }: StartDevServerOptions): Promise<void> {
-  // In order to type-check a single file, a temporary tsconfig
+  const { devCacheDir = join(workPath, '.vercel/cache') } = meta;
```

Comment:
```suggestion
  const { devCacheDir = join(workPath, '.vercel', 'cache') } = meta;
```

Reply:
`join()` in Windows normalizes it properly, but sure, why not.